management/user.go

```diff
@@ -343,3 +343,19 @@ func (m *UserManager) Blocks(id string, opts ...RequestOption) ([]*UserBlock, er
 func (m *UserManager) Unblock(id string, opts ...RequestOption) error {
 	return m.Request("DELETE", m.URI("user-blocks", id), nil, opts...)
 }
+
+// Link links two user accounts together forming a primary and secondary relationship.
+//
+// See: https://auth0.com/docs/api/management/v2#!/Users/post_identities
+func (m *UserManager) Link(id string, il *IdentityLink, opts ...RequestOption) (err error) {
+	return m.Request("POST", m.URI("users", id, "identities"), il, opts...)
+}
+
+// IdentityLink contains the data needed for linking an identity to a given user.
+//
+// Note: connection_id param is needed just to identify a particular database connection for the 'auth0' provider.
+type IdentityLink struct {
```

Comment:
Perhaps we should use the `User...` prefix to maintain the convention.

Reply:
Also maybe move the type definition above the methods to be in line with the rest of the code.